back/CinemaReservation.BusinessLayer/Models/AuthorizationResponseModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace CinemaReservation.BusinessLayer.Models
+{
+    public class AuthorizationResponseModel
+    {
+        public AuthorizationResponseModel(
+            int id,
+            string name,
+            string surname,
+            string email,
+            bool isAdmin
+        )
+        {
+            Id = id;
+            Name = name;
+            Surname = surname;
+            Email = email;
+            IsAdmin = isAdmin;
+        }
+        public int Id { get; set; }
```

Comment:
Properties should be placed before constructor. Also I recommend to lines separator between logical parts